setup.md

```diff
@@ -15,7 +15,7 @@ body_id: setup
 
 The first step is to make sure your application is appropriate for Cage. Does it meet the following criteria?
 
-* Multiple Dockerized services, each with their own repo on GitHub
+* Multiple Dockerized services
```

Comment:
We now support everything in the main repo! This makes `cage` usable for many more projects.